### DISCUSSION TITLE
Is that one can be used in React Native app?

### DISCUSSION THREAD

**Discussion by MasonTyTran** [matter-js/matter.js#549] [Q&A] [answered]
I'd like to know is this package made for creating matter device - embedded device or creating any application that can interact to matter enabled device?

**Comment by Apollon77** [comment, marked as answer]
In fact matter.js is a modular approach. Matter.js is the Pure Javascript "no native dependencies" package with the main logic.

In order to really work you needto implement "Platform near" features like UDP/network (mainly), potentially also BLE and some other, for the respective platform. Full information see https://github.com/project-chip/matter.js#how-to-use-matterjs-in-own-projects

We currently have implemented these parts for node.js. For react native you would need to implement them for the relevant react-native environment. If you like to do this we are happy to include such a package - like the nodejs package - also into the repository.

**Comment by MasonTyTran** [comment]
That's exactly what I've been understanding. Thank u for your confirmation
